### PR TITLE
Providing additional parameter for info message

### PR DESCRIPTION
**Pull request by rockstardev**
Resolves: #5755

Here is how it looks after this slight change

https://github.com/btcpayserver/btcpayserver/assets/5191402/08ac47da-4e91-4e25-99eb-73c32d64ccab

### CODE REVIEW

**Comment by pavlenex** [comment]
cACK, sweet addition!

**Comment by dennisreimann** [comment]
Code looks good. Only thought or sidenote: As this would allow providing links to the send page with custom messages I'm thinking if there might be ways to abuse this and trick someone.

**Comment by rockstardev** [comment]
@dennisreimann I've used `Message` in `StatusMessageModel` and not `Html`, exactly to prevent "richly formatted" messages. 

Also, you previously could already pass in strings using `message` property of `bip21`... so you could've already used that to abuse/trick people.

**Comment by dstrukt** [comment]
🔥 looks great

**Review by Kukks — COMMENTED** [review]
Can't you do TempData.SetStatusMessageModel in your own code  before the redirect? I  believe that Tempdata persists between 1 internal redirect?

**Comment by rockstardev** [comment]
Code updated as we agreed on the meeting, per @Kukks feedback.

**Review by dennisreimann — APPROVED** [review]
ACK. Yet this in an indicator for us to refactor the temp message handling after this PR.

File: BTCPayServer/Controllers/UIWalletsController.cs [hunk 1]
```diff
@@ -539,7 +540,7 @@ private async Task<bool> CanUseHotWallet()
                 }
                 catch (Exception ex) { model.RateError = ex.Message; }
             }
-
+
```
Comment (Kukks) [inline BTCPayServer/Controllers/UIWalletsController.cs:543]:
```suggestion

```

**Review by Kukks — APPROVED** [review]

**Pull request merged**